src/components/typeahead/typeahead.vue

```diff
@@ -265,6 +266,7 @@
     input {
         outline: 0;
         flex: 1;
+        margin: 7px 0;
```

Comment:
Input size should match other inputs; content should be smaller
From Design
"Initial input field to be the same, smaller content. Can we have the fields 40px height, the tag-like-content within a 30px height container?"